src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs

```diff
@@ -75,17 +78,28 @@ private void OnRequestShowFileDialog(object sender, PackagePathEventArgs e)
             
             e.Cancel = true;
 
-            var dialog = new DynamoFolderBrowserDialog
+            // Handle for the case, initialPath does not exist.
+            var errorCannotCreateFolder = PathHelper.CreateFolderIfNotExist(initialPath);
+            if (errorCannotCreateFolder == null)
             {
-                // Navigate to initial folder.
-                SelectedPath = initialPath,
-                Owner = this
-            };
+                var dialog = new DynamoFolderBrowserDialog
+                {
+                    // Navigate to initial folder.
+                    SelectedPath = initialPath,
+                    Owner = this
+                };
 
-            if (dialog.ShowDialog() == System.Windows.Forms.DialogResult.OK)
+                if (dialog.ShowDialog() == System.Windows.Forms.DialogResult.OK)
+                {
+                    e.Cancel = false;
+                    e.Path = dialog.SelectedPath;
+                }
+
+            }
+            else
             {
-                e.Cancel = false;
-                e.Path = dialog.SelectedPath;
+                string errorMessage = string.Format(Wpf.Properties.Resources.PackageFolderNotAccessible, "\n", initialPath, "\n");
```

Comment:
You actually can put enter in resource string, so it is not necessary to format string to just add enter.